tests/RobotFramework/tests/cumulocity/registration/device_registration.robot

```diff
@@ -193,13 +198,55 @@ Entities persisted and restored
         Should Have MQTT Messages    c8y/s/us/plc2    message_contains=102    date_from=${timestamp}    minimum=0    maximum=0
     END
 
+Entities send to cloud on restart
+    Execute Command    tedge mqtt pub --retain 'te/factory/shop/plc1/' '{"@type":"child-device","@id":"plc1"}'
```

Comment:
Could you align the `@id` values used in this test along the lines of https://github.com/thin-edge/thin-edge.io/pull/2657 ?